### ISSUE TITLE
Examples shown are not accessible. 

### ISSUE THREAD

**Issue by mlongoria** [MicrosoftDocs/powerbi-docs#1389]
While I understand you are showing off a feature, these examples are not accessible, mostly due to color contrast, and no one should try to emulate them. In the top image, you cannot read the charts because the background image is too bright and the charts are transparent. The examples on this page generally conflict the the recommendations provided on https://docs.microsoft.com/en-us/power-bi/visuals/power-bi-visualization-best-practices#principles-of-visual-design.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: db547626-10e8-5ea4-e1c3-7da6c236ce9f
* Version Independent ID: 4529c3a6-1f8f-2e8c-4eaf-9bde90c4df54
* Content: [Use visual elements to enhance Power BI reports - Power BI](https://docs.microsoft.com/en-us/power-bi/desktop-visual-elements-for-reports#feedback)
* Content Source: [powerbi-docs/desktop-visual-elements-for-reports.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/live/powerbi-docs/desktop-visual-elements-for-reports.md)
* Service: **powerbi**
* Sub-service: **powerbi-desktop**
* GitHub Login: @davidiseminger
* Microsoft Alias: **davidi**

**Comment by mihart** [comment]
Thanks for your comment, @mlongoria. This issue has been assigned to the doc author for review.

**Comment by davidiseminger** [comment]
Thank you for your feedback, we appreciate your input. We also strive to make our products accessible, even though some examples (like what you cited) may not be great examples of the best practices for accessibility. We'll continue to improve our samples and examples, always with accessibility in mind.
